x-pack/package.json

```diff
@@ -85,6 +103,7 @@
     "gulp-mocha": "2.2.0",
     "gulp-multi-process": "^1.3.1",
     "hapi": "^17.5.3",
+    "husky": "^0.14.3",
```

Comment:
We don't use husky anymore, you should be able to remove it

Reply:
done